models/ad_reporting__ad_report.sql

```diff
@@ -10,7 +10,7 @@ with base as (
 aggregated as (
     
     select 
-        date_day,
+        date_day as date_day_test,
```

Comment:
Same comment as above here

Reply:
yeah this is a workaround for https://github.com/dbt-labs/dbt_metrics/issues/103 -- the fix will be in v0.4.0 of the metrics package